Framework/Crystal/inc/MantidCrystal/FindSXPeaks.h

```diff
@@ -114,6 +102,11 @@ class DLLExport FindSXPeaks : public API::Algorithm {
 
   //
   void reducePeakList(const peakvector &, Mantid::API::Progress &progress);
+
+  // Check if this workspace has units of TOF
+  FindSXPeaksHelper::XAxisUnit
+  workspaceHasTOFUnits(Mantid::API::MatrixWorkspace_const_sptr workspace) const;
```

Comment:
This method should probably have a different name now that it no longer returns a `bool`.